common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -223,7 +224,7 @@ def get_html(self):
                             sources.append(url)
                         if self.download_video:
                             # function returns None when the url cannot be re-written
-                            rewritten_link = rewrite_video_url(cdn_url, url)
+                            rewritten_link = rewrite_video_url(cdn_url, url, cdn_rewritable_source_domains)
```

Comment:
nit: now that we're up to 3 vars, I'd explicitly call them out when accessing the method (e.g., `whitelist=cdn_rewritable_source_domains`)